command/sub8_launch/launch/subsystems/cameras.launch

```diff
@@ -1,4 +1,4 @@
 <launch>
   <include file="$(find sub8_launch)/launch/subsystems/cameras/launch/front_cameras.launch" />
-  <!-- include file="$(find sub8_launch)/launch/subsystems/cameras/launch/down_cameras.launch" / -->
+  <include file="$(find sub8_launch)/launch/subsystems/cameras/launch/down_cameras.launch" />
```

Comment:
Make sure the hostname is used to connect to the camera as opposed to the IP address.